packages/cli/package.json

```diff
@@ -14,32 +14,32 @@
   "dependencies": {
     "@boostercloud/framework-core": "workspace:^3.0.0",
     "@boostercloud/framework-types": "workspace:^3.0.0",
-    "@oclif/core": "3.15.0",
-    "@oclif/plugin-help": "^5",
+    "@oclif/core": "4.2.7",
+    "@oclif/plugin-help": "^6.2.25",
     "chalk": "^2.4.2",
     "execa": "^2.0.3",
     "fp-ts": "^2.11.0",
     "fs-extra": "^8.1.0",
     "inflected": "2.1.0",
     "inquirer": "^7.0.0",
-    "mustache": "4.1.0",
+    "mustache": "4.2.0",
     "ora": "^3.4.0",
-    "ts-morph": "^19.0.0",
+    "ts-morph": "^25.0.1",
```

Comment:
does this upgrade introduce any breaking changes on it's API? It seems to be a major version upgrade